pilot/pkg/networking/plugin/plugin.go

```diff
@@ -32,6 +32,17 @@ const (
 	ListenerProtocolTCP
 	// ListenerProtocolHTTP is an HTTP listener.
 	ListenerProtocolHTTP
+
+	// Authn is the name of the authentication plugin passed through the command line
+	Authn = "authn"
+	// Authz is the name of the rbac plugin passed through the command line
+	Authz = "authz"
+	// Envoyfilter is the name of the envoyfilter plugin passed through the command line
+	Envoyfilter = "envoyfilter"
```

Comment:
what is envoyfilter?

Reply:
It's this one https://github.com/istio/istio/blob/513c991d0495f8cfef87aa90cd17739454326a67/pilot/pkg/networking/plugin/registry/registry.go#L34, is that not the correct name for it?

Reply:
I don't know, just asking as a drive-by comment. Can we rename it to something that actually means something?

Reply:
Sure, I'm open to suggestions. I was trying to keep the plugin names that come in through the command line arguments aligned with the package names to avoid confusion.

Reply:
It's for the EnvoyFilters CRD which allows inserting custom filters into xDS config. That's the correct name for it given the names in istio/api.